src/Algebra/Bundles.agda

```diff
@@ -670,6 +670,25 @@ record CommutativeSemiring c ℓ : Set (suc (c ⊔ ℓ)) where
     }
 
 
+record CommutativeCancellativeSemiring c ℓ : Set (suc (c ⊔ ℓ)) where
+  infixl 7 _*_
+  infixl 6 _+_
+  infix  4 _≈_
+  field
+    Carrier                           : Set c
+    _≈_                               : Rel Carrier ℓ
+    _+_                               : Op₂ Carrier
+    _*_                               : Op₂ Carrier
+    0#                                : Carrier
+    1#                                : Carrier
+    isCommutativeCancellativeSemiring : IsCommutativeCancellativeSemiring
+                                          _≈_ _+_ _*_ 0# 1#
+
+  open IsCommutativeCancellativeSemiring isCommutativeCancellativeSemiring public
+  
+CCancellativeSemiring = CommutativeCancellativeSemiring  -- a shorthand
```

Comment:
We don't have shorthands for anything else in this file and I don't the proliferation of aliases helps. Please remove.

Reply:
Please, consider my commit of October 8.
All your current notes are implemented.